doc/classes/AnimationNode.xml

```diff
@@ -244,5 +244,11 @@
 		<constant name="FILTER_BLEND" value="3" enum="FilterAction">
 			Paths matching the filter will be blended (by the blend value).
 		</constant>
+		<constant name="NOTIFY_STARTED" value="0" enum="NotifyReason">
+			The node has started playback.
+		</constant>
+		<constant name="NOTIFY_FINISHED" value="1" enum="NotifyReason">
+			The node has finished playback.
+		</constant>
```

Comment:
Using ENUM is the right direction to go. However, for consistency with something similar to this that already exists in Godot, we recommend the signal name NOTIFICATION_XXX instead of NOTIFY_XXX.

Reply:
Hm, would `NOTIFICATION_XYZ` cause confusion with the existing notification stuff? I preemptively changed the enum name as well but that causes the c# build to complain about hiding `Notification` so that at least can't be shared. Maybe `EVENT_STARTED` etc with enum name `Event`?

Reply:
How about `ANIMATION_NODE_NOTIFICATION_XXX` ?